cost-snapshots/default/CostOfEverything-CryptoGetAccountInfoHappyPath-costs.properties

```diff
@@ -0,0 +1,11 @@
+#Cost snapshot
```

Comment:
Unclear how this file and mainnet-throttles.json got added to this pr when I did a sync with develop

Reply:
unclear on how to do that without deleting from upstream.  since there are no conflicts with develop, I propose I just leave them in.

Reply:
I think that these two files got _deleted_ in `develop` but they were still in your branch working tree or something and git thought you _added_ them yourself.  This just happened to me in the platform part of the repo.  If you pull develop and `find . -name 'mainnet-thr*'` you won't see anything.  (I didn't track down the commit where it was deleted though I'm sure it's there.)